### PR TITLE
feat: show error when user is not from Rootstrap

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:

[Ver el tema de acceso solo con email de rootstrap](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Ver%20el%20tema%20de%20acceso%20solo%20con%20email%20de%20rootstrap)

## What & Why:

This PR adds a new page to redirect users that entered with an email that is not `@rootstrap.com`. These user's don't have access to the db (because of a rule I added), and so they can't play the wordle game. To help them notice what is wrong, now I redirect them to a new `Invalid User` page that only contains a text explaining and a Logout Button. 

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,19 +12,28 @@ import './App.css';
 
 function App() {
   const t = useTranslation();
-  const { authenticated } = useAuth();
+  const { authenticated, user } = useAuth();
+
+  const { email = '' } = user || {};
```
Comment (gastonri) [inline src/App.js:17]: WDYT moving this logic to a hook? Let's say `useRootstrapAuth` or something like that?

File: src/App.js [hunk 2]
```diff
@@ -12,19 +12,28 @@ import './App.css';
...
+  const emailDomain = email.split('@')[1];
+  const isRootstrapDomain = emailDomain === 'rootstrap.com';
```
Comment (gastonri) [inline src/App.js]: WDYT moving this logic to a hook? Let's say useRootstrapAuth or something like that?

File: src/pages/InvalidUser/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  const { user } = useAuth();
+
+  const { email, name: completeName } = user || {};
+
+  const name = completeName?.split(' ')[0];
+
+  return (
+    <div className="invalid-user">
+      <p className="invalid-user-message">
+        {`Lo siento ${name} pero tu email "${email}" no pertenece a Rootstrap.`}
```
Comment (gastonri) [inline src/pages/InvalidUser/index.js:16]: What do you think about implementing a library for internalization? Or at least a file that has all the verbiage together

**Review by gastonri — COMMENTED** [review]
Everything looks good! I left two comments with possible improvements 🙂

File: src/pages/InvalidUser/index.js (hunk 3 above)
Comment (ManuViola77) [reply src/pages/InvalidUser/index.js:16]: Yes, I have that pending. I haven't done it yet 😅  But I know I should change it. In another PR I will localize all texts (they are a lot 😅 )

File: src/App.js (hunk 2 above)
Comment (ManuViola77) [reply src/App.js]: done 😄

**Review by gastonri — APPROVED** [review]
🚀

**Pull request merged**